docs/API-functions.md

```diff
@@ -1144,8 +1144,8 @@ typedef struct SRT_MsgCtrl_
    int flags;            // Left for future
    int msgttl;           // TTL for a message, default -1 (no TTL limitation)
    int inorder;          // Whether a message is allowed to supersede partially lost one. Unused in stream and live mode.
-   int boundary;         //0:mid pkt, 1(01b):end of frame, 2(11b):complete frame, 3(10b): start of frame
-   uint64_t srctime;     // source timestamp (usec), 0: use internal time     
+   int boundary;         // 0:mid pkt, 1(01b):end of frame, 2(11b):complete frame, 3(10b): start of frame
+   int64_t srctime;      // source time in mic(usec), 0: use internal time     
```

Comment:
What is "mic"? I think it should say something like: "source time (usec since steady clock epoch)". It is extremely important that users do not specify here the time read from `gettimeofday` because it make work for them most of the time making a false impression that their code is correct.

Reply:
Fixed